### PR TITLE
Sample_flows

### PR DESCRIPTION
**Pull request by giorgiozoppi**
This PR create a new directory sample for early adapters and new developer onboarding.

### CODE REVIEW

**Comment by giorgiozoppi** [comment]
I added a sample directory to make it easy the adoption of this code.

File: samples/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+all: compile 
+clean:
+	@rm flow
+setup:
+	@sudo docker run -p 6379:6379 -d redis
```
Comment (s8sg) [inline samples/Makefile:5]: lets remove sudo

File: Makefile [hunk 2]
```diff
@@ -7,7 +7,6 @@ coverage: | __go-pkg-list
 	@go test -gcflags=-l -v ${GO_PKG_LIST} -coverprofile /tmp/pls_cp.out || true
 	@go tool cover -html=/tmp/pls_cp.out -o /tmp/coverage.html
 	@echo "You can find coverage report at /tmp/coverage.html"
-
```
Comment (s8sg) [inline Makefile:10]: Lets remove unnecessary changes

File: samples/Makefile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+all: compile
```
Comment (s8sg) [inline samples/Makefile:1]: Can you also add a README page in the sample section.

File: samples/Makefile (hunk 1 above)
Comment (giorgiozoppi) [reply samples/Makefile:5]: done

File: Makefile (hunk 2 above)
Comment (giorgiozoppi) [reply Makefile:10]: done

File: samples/Makefile (hunk 3 above)
Comment (giorgiozoppi) [reply samples/Makefile:1]: ok.

**Comment by giorgiozoppi** [comment]
@s8sg  please approve.

**Review by s8sg — DISMISSED** [review]

**Review by s8sg — APPROVED** [review]
